build-status.el

```diff
@@ -375,8 +375,7 @@ Signals an error if the response does not contain an HTTP 200 status code."
 ;;;###autoload
 (define-minor-mode build-status-mode
   "Monitor the build status of the buffer's project."
-  :global t
```

Comment:
Seems remove `:global t` is not a solution. I think you should cherry-pick my commit to fix the package `provide` name at first for hot fix ELPA package.